src/drivers/imu/bosch/bmi088/BMI088_Accelerometer.cpp

```diff
@@ -43,13 +43,14 @@ namespace Bosch::BMI088::Accelerometer
 BMI088_Accelerometer::BMI088_Accelerometer(I2CSPIBusOption bus_option, int bus, uint32_t device, enum Rotation rotation,
 		int bus_frequency, spi_mode_e spi_mode, spi_drdy_gpio_t drdy_gpio) :
 	BMI088(DRV_ACC_DEVTYPE_BMI088, "BMI088_Accelerometer", bus_option, bus, device, spi_mode, bus_frequency, drdy_gpio),
+	ModuleParams(nullptr),
 	_px4_accel(get_device_id(), ORB_PRIO_HIGH, rotation)
 {
 	if (drdy_gpio != 0) {
 		_drdy_interval_perf = perf_alloc(PC_INTERVAL, MODULE_NAME"_accel: DRDY interval");
 	}
 
-	ConfigureSampleRate();
+	ConfigureSampleRate(_param_imu_gyro_rate_max.get());
```

Comment:
FYI this was left fixed for the sake of aligning the integration. If you look at `vehicle_imu.delta_velocity_dt` you'll see occasional spikes of the effective sample rate (IMU_GYRO_RATEMAX rounded to BMI088 accel ODR). I had considered always running the accel at max (1600 Hz), but it didn't seem worth the cpu.

I think it's fine (and still much better than it was before), just something to be aware of. I also found that the clocks seems to be slightly different between the accel & gyro (larger difference than between 2 separate Invensense IMUs).

![Screenshot from 2020-06-11 12-16-35](https://user-images.githubusercontent.com/84712/84412575-73f85c80-abdd-11ea-8db7-56025c75efa0.png)